resources/js/app.js

```diff
@@ -2,13 +2,30 @@
  * Here, we include all of our external dependencies
  */
 import {Notyf} from 'notyf';
+import {createApp} from 'vue';
+import NotificationList from "../vue/components/NotificationList.vue";
+import NotificationBell from "../vue/components/NotificationBell.vue";
 
 require("./bootstrap");
 require("awesomplete/awesomplete");
 require("leaflet/dist/leaflet.js");
 require("./api/api");
 require("./components/maps");
 
+const app = createApp({
+	data() {
+		return{
+			count: 0
+		}
+	}
+});
+
+app.component('notificationlist', NotificationList);
+app.component('notificationbell', NotificationBell);
```

Comment:
I might be wrong, but you're using these components in DOM templates so shouldn't these be registered as `NotificationList` and `NotificationBell` and then be referenced in _app.blade.php_ as `<notification-list>` and `<notification-bell>` respectively?

https://vuejs.org/guide/components/registration.html#component-name-casing

Reply:
afaik Vue resolves to every possible case-constellation, but kebab-case seems to be the "correct" way of doing it. Will do!